### PR TITLE
[SW-2619] Make Maximum Size of Requests and Responses on Flow UI Proxy Configurable

### PR DESCRIPTION
**Pull request by mn-mikke**
This PR makes the limits configurable and also changes the default maximum request size from 4kB to 32 kB and the default maximum request size from 16kB to 32 kB. This change fixes Flow UI accessibility  on Databricks Azure cluster when `spark.databricks.passthrough.enabled` is set to `true`.

### CODE REVIEW

File: core/src/main/scala/water/webserver/jetty9/SparklingWaterJettyHelper.scala [hunk 1]
```diff
@@ -60,8 +60,10 @@ class SparklingWaterJettyHelper(hc: H2OContext, conf: H2OConf, h2oHttpView: H2OH
     handler.addServletMapping(m)
     val ipPort = conf.h2oCluster.get
     holder.setInitParameter("proxyTo", s"${conf.getScheme()}://$ipPort${conf.contextPath.getOrElse("")}")
-    holder.setInitParameter("idleTimeout", s"${conf.restApiTimeout}")
-    holder.setInitParameter("timeout", s"${conf.restApiTimeout}")
+    holder.setInitParameter("idleTimeout", conf.restApiTimeout.toString)
+    holder.setInitParameter("timeout", conf.restApiTimeout.toString)
+    holder.setInitParameter("requestBufferSize", conf.flowProxyRequestMaxSize().toString)
+    holder.setInitParameter("responseBufferSize", conf.flowProxyRequestMaxSize().toString)
```
Comment (michalkurka) [inline core/src/main/scala/water/webserver/jetty9/SparklingWaterJettyHelper.scala:66]: looks like a typo? should be responseMaxSize?

**Comment by michalkurka** [comment]
What about the head max size? In H2O we do this


    connector.setRequestHeaderSize(getSysPropInt(proto+".requestHeaderSize", 32*1024));
    connector.setRequestBufferSize(getSysPropInt(proto+".requestBufferSize", 32*1024));
    connector.setResponseHeaderSize(getSysPropInt(proto+".responseHeaderSize", connector.getResponseHeaderSize()));
    connector.setResponseBufferSize(getSysPropInt(proto+".responseBufferSize", connector.getResponseBufferSize()));

File: core/src/main/scala/water/webserver/jetty9/SparklingWaterJettyHelper.scala (hunk 1 above)
Comment (mn-mikke) [reply core/src/main/scala/water/webserver/jetty9/SparklingWaterJettyHelper.scala:66]: Good catch, thanks!

**Comment by mn-mikke** [comment]
@michalkurka Changed the default values to 32kB, thanks!

**Review by koniecsveta — APPROVED** [review]

**Pull request merged**